### PR TITLE
admin: Resources on ML models, integration methods and overall techstack

### PR DESCRIPTION
**Pull request by rupali-p**
Trello Tickets T011 (Find Resources for the ML Models) and T012 (Find Resources for integrating a Front-End with the Data Model): 
- overview of the different technologies we can work with

### CODE REVIEW

**Comment by ClodaghMM** [comment]
Great stuff, thank you for doing that @rupali-p . My only question is should we be noting down the disadvantages of some of the frameworks you've mentioned? All I can see are advantages. 

All good if that's not the case but figured to raise the question.

**Review by tam-debug — COMMENTED** [review]
This is great, although when are we looking to finalise the development framework?

File: administration/rupali/Learning-ML.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+### Step 2: Choose a Development Framework / Loading it in
+
+Decide on the development framework for our web or mobile app. Popular choices include:
+For web apps: JavaScript (with libraries like TensorFlow.js or ONNX.js), Python (with web frameworks like Flask or Django for server-side integration).
+For mobile apps: Swift for iOS development and Kotlin or Java for Android development.
+
+In our web or mobile app, we can load the pre-trained machine learning model into memory using the framework-specific APIs. For example, in TensorFlow.js, we can use tf.loadGraphModel() to load a TensorFlow model.
+
+#### Streamlit:
```
Comment (ClodaghMM) [inline administration/rupali/Learning-ML.md:39]: Just adding a line here to refer to my previous comment on disadvantages (feel free to disregard if need be)

**Comment by ClodaghMM** [comment]
@rupali-p Just swinging back to this pr, were you planning to address Tam's and I comments or is this good to be merged in?

**Comment by rupali-p** [comment]
^ @ClodaghMM yep we discussed them during our standup last week Monday as I did a showcase for the demo - if you don't think it's been adequately addressed, I can explicitly add that in 
thanks

**Comment by ClodaghMM** [comment]
> ^ @ClodaghMM yep we discussed them during our standup last week Monday as I did a showcase for the demo - if you don't think it's been adequately addressed, I can explicitly add that in
> thanks

Ok, if we've addressed it then I think it should be good to go.

**Review by ClodaghMM — APPROVED** [review]
Approved, this is good to go as we've done a demo and clarified some concepts.

**Comment by rupali-p** [comment]
> > ^ @ClodaghMM yep we discussed them during our standup last week Monday as I did a showcase for the demo - if you don't think it's been adequately addressed, I can explicitly add that in
> > thanks
> 
> Ok, if we've addressed it then I think it should be good to go.

I can add more info if you want tho :DD AHAHA 
literally, just let me know bestie

**Review by tam-debug — APPROVED** [review]

**Review by JasmineBautista — APPROVED** [review]

**Pull request merged**